python-shell/src/gafferpy_examples/__init__.py

```diff
@@ -0,0 +1,11 @@
+__version__ = "2.0.0-alpha-0.3"
+
+__title__ = "gafferpy"
+__description__ = "Gaffer Python Shell"
+__uri__ = "https://github.com/gchq/gafferpy-tools"
+__doc__ = __description__ + " <" + __uri__ + ">"
+
+__author__ = "GCHQ"
+
+__license__ = "Apache 2.0"
+__copyright__ = "Crown Copyright (c) 2016-2017"
```

Comment:
Needs an update?

Reply:
Yeah these have all never been changed so I left them, but we should probably be updating dates in READMEs and __init__.py files as tedious as it is